includes/Integrations/AMP.php

```diff
@@ -162,8 +163,15 @@ public function add_amp_content_sanitizers( $sanitizers ) {
 
 		$story = new Story();
 		$story->load_from_post( $post );
+
+		$publisher_logo_id  = get_post_meta( $post->ID, Story_Post_Type::PUBLISHER_LOGO_META_KEY, true );
+		$publisher_logo_url = wp_get_attachment_image_url( $publisher_logo_id, Image_Sizes::PUBLISHER_LOGO_IMAGE_SIZE );
+
+		/* This filter is documented in includes/Discovery.php */
+		$publisher_logo_url = apply_filters( 'web_stories_publisher_logo', $publisher_logo_url, $post );
```

Comment:
Also of note here, why `deprecated` the placeholder? Why not just add a new param like WordPress core does? There is a backward compatibility issue here.

Reply:
The second param was never really used and removed a long time ago. We don't need to hold on to that forever like core does.